data/sqlite3/account_store.go

```diff
@@ -57,10 +57,11 @@ func (db *AccountStore) Create(u string, p []byte) (*models.Account, error) {
 		PasswordChangedAt: now,
 		CreatedAt:         now,
 		UpdatedAt:         now,
+		LastLoginAt:       now,
```

Comment:
can you say more about why this was needed for sqlite3 but not mysql or postgresql?

Reply:
Ops, my bad, the correct is mysql and postgresql way. I need only change to `LastLoginAt` to work with nil value, like `DeletedAt`.